semver.py

```diff
@@ -433,7 +442,7 @@ def _compare_by_keys(d1, d2):
 
 def compare(ver1, ver2):
     """
-    Compare two versions.
+    Compare two versions strings.
 
     :param ver1: version string 1
```

Comment:
A quick question / suggestion - maybe in such case we should also add typing? Like for instance: 
```suggestion
    :param str ver1: version string 1
```
IDEs (like PyCharm) would love that feature :)

Reply:
Generally you are right. :+1:
However, I'm not sure. We have already issue #213 which is targeted for major version 3. So maybe, I'd suggest to wait until we can release the next major release? Would that be ok?

Reply:
Yeah, sure :)